index.html

```diff
@@ -2388,9 +2389,13 @@ <h2 id="AudioParam">
               "#widl-AudioParam-exponentialRampToValueAtTime-void-float-value-double-endTime">
               exponentialRampToValueAtTime</a></code>, respectively.
             </p>
+            <p>
+              This method returns the current <a>AudioParam</a> object for easy
```

Comment:
I didn't find anything about AudioNode connect() allowing easy chaining in the spec (did I miss it?), so I don't see why we need to say anything about easy chaining.  I think it's fairly obvious you can chain them together now.